src/Php80/Resources/stubs/Attribute.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+#[Attribute(Attribute::TARGET_CLASS)]
+final class Attribute {
+    const TARGET_CLASS = 1;
+    const TARGET_FUNCTION = 2;
+    const TARGET_METHOD = 4;
+    const TARGET_PROPERTY = 8;
+    const TARGET_CLASS_CONSTANT = 16;
+    const TARGET_PARAMETER = 32;
+    const TARGET_ALL = 63;
+    const IS_REPEATABLE = 64;
+
+    /** @var int */
+    public $flags;
```

Comment:
This has a real type signature in php 8.0, but that's impractical to polyfill while supporting php versions older than php 7.4 and probably not going to matter for working php 8 applications

```
php > $x = new Attribute();
php > $x->flags = 'test';

Warning: Uncaught TypeError: Cannot assign string to property Attribute::$flags of type int in php shell code:1
Stack trace:
#0 {main}
  thrown in php shell code on line 1
```